pkgs/development/python-modules/ffcv/default.nix

```diff
@@ -0,0 +1,64 @@
+{ assertpy
+, buildPythonPackage
+, fetchFromGitHub
+, lib
+, libjpeg
+, numba
+, opencv4
+, pandas
+, pkgconfig
+, psutil
+, pytestCheckHook
+, pytorch-pfn-extras
+, terminaltables
+, torchvision
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "ffcv";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "libffcv";
+    repo = pname;
+    # See https://github.com/libffcv/ffcv/issues/158.
+    rev = "131d56235eca3f1497bb84eeaec82c3434ef25d8";
+    sha256 = "0f7q2x48lknnf98mqaa35my05qwvdgv0h8l9lpagdw6yhx0a6p2x";
+  };
+
+  # See https://github.com/libffcv/ffcv/issues/159.
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "'assertpy'," "" \
+      --replace "'fastargs'," "" \
+      --replace "'imgcat'," "" \
+      --replace "'matplotlib'," "" \
+      --replace "'psutil'," "" \
+      --replace "'sklearn'," "" \
+      --replace "'webdataset'," ""
+  '';
+
+  buildInputs = [ pkgconfig libjpeg ];
```

Comment:
I am not sure if pkgconfig the python package should go to nativeBuildInputs.

Reply:
Do you mean that I should move it to nativeBuildInputs? Or something else?

Reply:
Yeah, I think we want to move pkgconfig into nativeBuildInputs.

Reply:
Moving `pkgconfig` into `nativeBuildInputs` doesn't work unfortunately:
```
Executing setuptoolsBuildPhase
Traceback (most recent call last):
  File "/build/source/nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 30, in <module>
    extension_kwargs = pkgconfig('opencv4', extension_kwargs)
  File "setup.py", line 18, in pkgconfig
    raise Exception()
Exception
builder for '/nix/store/zsvvs4ispid5s6viwnpvcfl12p3hmzvl-python3.9-ffcv-0.0.3.drv' failed with exit code 1
```